src/app/shared/constants/cities.constant.json

```diff
@@ -0,0 +1,1544 @@
+[
```

Comment:
We have too much mock data on the UI side. It can affect on bundle size. I suggest using only the first 10 values and provide this data as constant instead of JSON: export const CitiesData = [ ... ]